### PR TITLE
MIN-72 Upgrade iOS SDK to v3.0.0

### PR DESCRIPTION
**Pull request by habib-minkasu**

### CODE REVIEW

File: iOS/Objective-C/SampleAppWKWebView/README.md [hunk 1]
```diff
@@ -85,14 +85,22 @@ mkcolorTheme.darkModeButtonTextColor = UIColor.whiteColor;
...
+    config.delegate = self;            // Minkasu2FACallbackDelegate
+    config._id = <merchant_id>;
+    config.token = <merchant_access_token>;
     config.merchantCustomerId =<merchant_customer_id>;
     //add customer to the Config object
     config.customerInfo = customer;
 
     Minkasu2FAOrderInfo *orderInfo = [Minkasu2FAOrderInfo new];
     orderInfo.orderId = <order_id>;
+    orderInfo.billingCategory = <billing_category>; // e.g. “FLIGHTS”
```
Comment (habib-minkasu) [inline iOS/Objective-C/SampleAppWKWebView/README.md:97]: Add comment similar to android

File: iOS/Objective-C/SampleAppWKWebView/README.md [hunk 2]
```diff
@@ -116,15 +124,44 @@ mkcolorTheme.darkModeButtonTextColor = UIColor.whiteColor;
...
     config.sdkMode = MINKASU2FA_SANDBOX_MODE;
 
+    NSError *error = nil;
     //Initializing Minkasu2FA SDK with WKWebView object
-    [Minkasu2FA initWithWKWebView:_wkWebView andConfiguration:config];
+    BOOL result = [Minkasu2FA initWithWKWebView:_wkWebView andConfiguration:config error:&error];
+    if (result) {
+        //Minkasu init success
+    } else {
+        //Minkasu init failed - handle error
```
Comment (habib-minkasu) [inline iOS/Objective-C/SampleAppWKWebView/README.md:133]: change to NSError

File: iOS/Objective-C/SampleAppWKWebView/README.md [hunk 3]
```diff
@@ -116,15 +124,44 @@ mkcolorTheme.darkModeButtonTextColor = UIColor.whiteColor;
...
+    }
 }
 ```
 
 - Make sure that your merchant_access_token and merchant_id are correct.
 - merchant_customer_id is a unique id associated with the currently logged in user
 - Initialize the SDK by calling ```[self initMinkasu2FA];``` before the Payment is initiated.
 
+## Implementing Minkasu2FA Callback Delegate
```
Comment (habib-minkasu) [inline iOS/Objective-C/SampleAppWKWebView/README.md:142]: Minkasu2FA Callback Delegate to Minkasu2FACallbackDelegate

File: iOS/Objective-C/SampleAppWKWebView/SampleAppWKWebView/ViewController.m [hunk 4]
```diff
@@ -201,4 +215,15 @@ - (IBAction)clickMenuOption:(id)sender {
     }
 }
 
+#pragma mark - Minkasu2FA Callback Delegate
```
Comment (habib-minkasu) [inline iOS/Objective-C/SampleAppWKWebView/SampleAppWKWebView/ViewController.m:218]: Minkasu2FACallback

File: iOS/Swift/SampleAppWKWebView-Swift/README.md [hunk 5]
```diff
@@ -128,14 +143,40 @@ func initMinkasu2FA(){
...
+## Implementing Minkasu2FA Callback Delegate
+
+1.Conforming to Minkasu2FACallbackDelegate Protocol on your ViewController class which initialize Minkasu2FA iOS SDK. ```Minkasu2FACallbackDelegate```
+2.Setting the delegate in Minkasu2FA config. ``` config.delegate = self```
+3.Implementing Minkasu2FACallback delegate method.
+
+```Swift
+func minkasu2FACallback(_ minkasu2FACallbackInfo: Minkasu2FACallbackInfo) {
+    if minkasu2FACallbackInfo.infoType == 1{  // INFO_TYPE_RESULT
+        // Refer readme file for minkasu2FACallbackInfo details.
```
Comment (habib-minkasu) [inline iOS/Swift/SampleAppWKWebView-Swift/README.md:167]: refer to table below

**Review by sachinminkasu — APPROVED** [review]

**Review by naveend77 — APPROVED** [review]

**Review by sachinminkasu — APPROVED** [review]

**Pull request merged**